tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -369,21 +371,20 @@ public function setUpScenario(BeforeScenarioScope $scope): void {
 	 *
 	 * @throws Exception|GuzzleException
 	 */
-	public function cleanDataAfterTests(): void
-	{
+	public function cleanDataAfterTests(): void {
 		$this->deleteAllSpacesOfTheType('project');
 	}
 
 	/**
 	 * The method first disables and then deletes spaces
+	 *
 	 * @param  string $driveType
 	 *
 	 * @return void
 	 *
 	 * @throws Exception|GuzzleException
 	 */
-	public function deleteAllSpacesOfTheType(string $driveType): void
-	{
+	public function deleteAllSpacesOfTheType(string $driveType): void {
```

Comment:
don't we run PHP lint here? this should have been caught by CI

Reply:
I ran `make test-php-style-fix` and got `No fixable errors were found`